### PR TITLE
Add November newsletter

### PR DESCRIPTION
**Pull request by ericholscher**
----
📚 Documentation preview 📚: https://readthedocs-about--334.org.readthedocs.build/

### CODE REVIEW

File: content/posts/newsletter-november-2024.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+tags: newsletter, community
+authors: Eric Holscher
+status: published
+image: /images/november-2024.jpg
+image_credit: Photo by <a href="https://unsplash.com/@hermanndoyo?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Doyo Hermann</a> on <a href="https://unsplash.com/photos/a-scenic-view-of-a-town-surrounded-by-mountains-y1VQvwSqYOE?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+
+## News and updates
+
+* The new **Community dashboard** at https://app.readthedocs.org will be made the default dashboard on November 12. We are still beta testing the Business dashboard, as it effects documentation authentication and requires a bit more testing. We expect to make the Business dashboard the default soon.
```
Comment (humitos) [inline content/posts/newsletter-november-2024.md:14]: 💯

File: content/posts/newsletter-november-2024.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community
+authors: Eric Holscher
+status: published
+image: /images/november-2024.jpg
+image_credit: Photo by <a href="https://unsplash.com/@hermanndoyo?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Doyo Hermann</a> on <a href="https://unsplash.com/photos/a-scenic-view-of-a-town-surrounded-by-mountains-y1VQvwSqYOE?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+
+## News and updates
```
Comment (humitos) [inline content/posts/newsletter-november-2024.md:12]: Now notification on latest and non-stable versions can be enabled/disabled independently.
https://github.com/readthedocs/readthedocs.org/pull/11744

Comment (humitos) [reply content/posts/newsletter-november-2024.md:12]: All the `build.tools` versions were updated.
https://github.com/readthedocs/readthedocs-docker-images/pull/209

**Review by humitos — APPROVED** [review]
Looks good to me. I left 2 extra changes we could include here as well.

File: content/posts/newsletter-november-2024.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+title: Read the Docs Newsletter - November 2024
+date: November 10, 2024
```
Comment (ericholscher) [inline content/posts/newsletter-november-2024.md:2]:
```suggestion
date: November 11, 2024
```

Will plan to ship it Monday.

File: content/posts/newsletter-november-2024.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+authors: Eric Holscher
+status: published
+image: /images/november-2024.jpg
+image_credit: Photo by <a href="https://unsplash.com/@hermanndoyo?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Doyo Hermann</a> on <a href="https://unsplash.com/photos/a-scenic-view-of-a-town-surrounded-by-mountains-y1VQvwSqYOE?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+
+## News and updates
+
+* The new **Community dashboard** at https://app.readthedocs.org will be made the default dashboard on November 12. We are still beta testing the Business dashboard, as it effects documentation authentication and requires a bit more testing. We expect to make the Business dashboard the default soon.
+* We've added **Multi-Factor Authentication (MFA)** to our platform, and it's now fully released. Give it a try to add an extra layer of security to your account.
```
Comment (ericholscher) [inline content/posts/newsletter-november-2024.md:15]:
```suggestion
* We've added **Multi-Factor Authentication (MFA)** to our platform, and it's now fully released. Give it a try to add an extra layer of security to your account.
* We have added **Python 3.13** as a build option, along with `updating other build tools <https://github.com/readthedocs/readthedocs.org/pull/11742>`_. 
```

File: content/posts/newsletter-november-2024.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+* We have added **Python 3.13** as a build option, along with `updating other build tools <https://github.com/readthedocs/readthedocs.org/pull/11742>`_. 
+* **Addons** continues to get iterative improvements, including allowing more granular configuration of documentation page documentations, and fixing bugs in the injection logic for large images.
```
Comment (agjohnson) [inline content/posts/newsletter-november-2024.md:17]: What are we referring to with `documentation page documentations`?

**Review by agjohnson — APPROVED** [review]
Looks good, just copy edits :+1:

File: content/posts/newsletter-november-2024.md (hunk 5 above)
Comment (ericholscher) [reply content/posts/newsletter-november-2024.md:17]: Ha, notifications 🙃 
```suggestion
* **Addons** continues to get iterative improvements, including allowing more granular configuration of `documentation notifications <https://docs.readthedocs.io/en/stable/doc-notifications.html>`_, and fixing bugs in the injection logic for large images.
```

**Pull request merged**